docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,72 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_prepare_model_MO_Python_API}
+
+- Model Optimizer has Python API for model conversion, which is represented by convert_model() method in openvino.tools.mo namespace.
+  convert_model() has all the functionality available from command line tool plus the possibility of passing Python objects (models, extensions) to convert_model() directly from memory.
```

Comment:
convert_model() has all the functionality available from command line tool plus the possibility of passing Python objects (models, extensions) to convert_model() directly from memory without leaving the training environment (Jupyter Notebook or training scripts).